src/styles/_header.scss

```diff
@@ -0,0 +1,67 @@
+@use './variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.img {
+  display: block;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: variables.$font-family-header;
+  font-size: variables.$title-font-size;
+  font-weight: variables.$title-font-weight;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    color: variables.$main-color;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__link:hover {
+    color: variables.$blue-color;
```

Comment:
The hover color for the navigation links should be `#00acdc` as specified in the task requirements. Ensure that the color variable used here matches this requirement.